src/main/java/com/conveyal/datatools/manager/jobs/DeployJob.java

```diff
@@ -180,15 +180,24 @@ public OtpServer getOtpServer() {
         return otpServer;
     }
 
+    /**
+     * Primary constructor for kicking off a deployment of transit/OSM/config data to OpenTripPlanner.
+     *
+     * FIXME: It appears that DeployType#Replace is the only type used in the non-test code. Should the others be
```

Comment:
I'm inclined to answer yes as I'm generally an advocate for removing unused code paths, but perhaps @landonreed may want to keep it around. I think there are still some UI settings for deploying with a prebuilt graph, but I'm not sure if there are endpoints to match.

Reply:
Yea, I think this comment was added by me. I'm having trouble finding where this might be in the UI. In any case, yea I'm in favor of removing as well...

Reply:
@landonreed before I remove this `DeployType.USE_PREBUILT_GRAPH` is referenced within `DeployJob` (line 581). So, there will be no pre-built graph and therefore a graph must be built with every deployment?

Reply:
This touches a lot of code, so actually I don't really think we should remove as part of this PR. Let's perhaps create an issue about this and handle separately.

Reply:
https://github.com/ibi-group/datatools-server/issues/366